<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diwan-catalogue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="bannerWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_banner">
       <item>
        <widget class="RotatableLabel" name="labelLogo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MarqueeLabel" name="labelAppName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Diwan-catalogue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_maison_precedente">
        <property name="text">
         <string>&lt;&lt; Maison Précédente</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_maison_suivante">
        <property name="text">
         <string>Maison Suivante &gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="content_layout" stretch="1,4,2">
      <item>
       <layout class="QVBoxLayout" name="thumbnail_layout">
        <item>
         <widget class="ClickableLabel" name="thumbnail_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>border: 2px solid transparent; border-radius: 5px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickableLabel" name="thumbnail_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>border: 2px solid transparent; border-radius: 5px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickableLabel" name="thumbnail_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>border: 2px solid transparent; border-radius: 5px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="photo_section_layout">
        <item>
         <widget class="QLabel" name="photo_display_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="text">
           <string>Photo</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="photo_nav_layout">
          <item>
           <widget class="QPushButton" name="pushButton_photo_precedente">
            <property name="text">
             <string>&lt; Photo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_photo_compteur">
            <property name="text">
             <string>Photo 0/0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_photo_suivante">
            <property name="text">
             <string>Photo &gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="details_and_description_layout">
        <item>
         <widget class="QGroupBox" name="detailsGroup">
          <property name="title">
           <string>Détails de la Maison</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_id_key">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_id_val">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_type_key">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_type_val">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_standing_key">
             <property name="text">
              <string>Standing</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_standing_val">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_chambres_key">
             <property name="text">
              <string>Chambres</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_chambres_val">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_toilettes_key">
             <property name="text">
              <string>Toilettes</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_toilettes_val">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="descriptionGroupBox">
          <property name="title">
           <string>Description</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTextBrowser" name="text_description"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBase">
    <property name="title">
     <string>Base</string>
    </property>
    <addaction name="actionCharger"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuMaison">
    <property name="title">
     <string>Maison</string>
    </property>
    <addaction name="actionRechercher"/>
    <addaction name="actionInserer"/>
    <addaction name="actionSupprimer"/>
    <addaction name="actionModifier"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <widget class="QMenu" name="menuFonction_de_Hachage">
     <property name="title">
      <string>Fonction de Hachage</string>
     </property>
     <addaction name="actionF1"/>
     <addaction name="actionF2"/>
     <addaction name="actionF3"/>
     <addaction name="separator"/>
     <addaction name="actionSans_Hashage"/>
    </widget>
    <addaction name="menuFonction_de_Hachage"/>
    <addaction name="separator"/>
    <addaction name="actionComparaison_Graphique"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
      <addaction name="action_video_presentation"/>
   <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuBase"/>
   <addaction name="menuMaison"/>
   <addaction name="menuOutils"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCharger">
   <property name="text">
    <string>Charger Données</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder Modifs</string>
   </property>
  </action>
  <action name="actionInserer">
   <property name="text">
    <string>Insérer...</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionModifier">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionRechercher">
   <property name="text">
    <string>Rechercher...</string>
   </property>
  </action>
  <action name="action_video_presentation">
   <property name="text">
    <string>Présentation Vidéo</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionComparaison_Graphique">
   <property name="text">
    <string>Comparaison Graphique</string>
   </property>
  </action>
  <action name="actionF1">
   <property name="text">
    <string>F1 - Somme Caractères</string>
   </property>
  </action>
  <action name="actionF2">
   <property name="text">
    <string>F2 - Hachage Polynomial</string>
   </property>
  </action>
  <action name="actionF3">
   <property name="text">
    <string>F3 - DJB2</string>
   </property>
  </action>
  <action name="actionSans_Hashage">
   <property name="text">
    <string>Sans Hashage</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RotatableLabel</class>
   <extends>QLabel</extends>
   <header>ui/rotatablelabel.h</header>
  </customwidget>
  <customwidget>
   <class>MarqueeLabel</class>
   <extends>QLabel</extends>
   <header>ui/marqueelabel.h</header>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ui/clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>